src/Endatix.WebHost/appsettings.Development.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=localhost;Integrated Security=true;Initial Catalog=Endatix.App;"
+    "DefaultConnection": "Server=localhost;Integrated Security=true;TrustServerCertificate=true;Initial Catalog=Endatix.App;"

```

Comment:
This is intentional, right?

Reply:
It is, but if we plan to use identical development connection strings we should coordinate on SQL Server versions and preferred SSL setup. Alternatively we could consider storing the dev connection string in a gitignored manner?

Reply:
Not a topic for now. Just a note to share in the Engineering group to discuss it. I am also using it locally.